zeeguu/api/endpoints/user_articles.py

```diff
@@ -94,3 +94,20 @@ def user_articles_cohort():
     """
 
     return json_result(flask.g.user.cohort_articles_for_user())
+
+# ---------------------------------------------------------------------------
+@api.route("/user_articles/foryou", methods=("GET",))
+# ---------------------------------------------------------------------------
+@cross_domain
+@with_session
+def user_articles_foryou():
+    article_infos = []
+    try:
+        articles = content_recommendations(flask.g.user.id, flask.g.user.learned_language_id)
+        print("Sending CB recommendations")
+    except:
+        #Usually no recommendations when the user has not liked any articles
```

Comment:
We don't want to silently catch all errors, make sure to at least log the exception if there is an error.

Currently we use Sentry to keep track of exceptions in zeeguu:

```
    except Exception as e:
        from sentry_sdk import capture_exception

        capture_exception(e)
```

Reply:
It should be ready now